client/verta/verta/utils.py

```diff
@@ -312,3 +314,27 @@ def predict(self, **kwargs):
         }
 
         return self.session.run(self.output_tensors, input_dict)
+
+
+def multiple_arguments_for_each(argument, name, action, get_keys, overwrite):
+    name = name
+    argument = list(map(lambda s: s.split('='), argument))
+    if argument and len(argument) > len(
+            set(map(lambda pair: pair[0], argument))):
+        raise click.BadParameter("cannot have duplicate {} keys".format(name))
+    if argument:
+        argument_keys = set(get_keys())
+
+        for pair in argument:
+            if len(pair) != 2:
+                raise click.BadParameter("key and path for {}s must be separated by a '='".format(name))
+            (key, _) = pair
+            if key == "model":
+                raise click.BadParameter("the key \"model\" is reserved for model")
+
+            if not overwrite and key in argument_keys:
+                raise click.BadParameter(
+                    "key \"{}\" already exists; consider using --overwrite flag".format(key))
+
+        for (key, path) in argument:
+            action(key, path)
```

Comment:
Please move this to [_internal_utils/_utils.py](https://github.com/VertaAI/modeldb/blob/master/client/verta/verta/_internal_utils/_utils.py) instead

Reply:
moved